lib/screens/HomeScreen.tsx

```diff
@@ -331,6 +332,16 @@ const HomeScreen: React.FC<HomeScreenProps> = ({ navigation, route }) => {
       padding: 10,
       alignSelf: "center",
     },
+    seachBar:{
+      backgroundColor: theme.homeColor,
+      borderRadius: 20,
+      fontSize: 18,
+      padding: 20,
+      margin: 20,
+      color: theme.text,
+      borderWidth: 3,
+    
+    },
```

Comment:
adding the colors to show importance of keeping the search bar intact when it comes to styling.